lib/engine.js

```diff
@@ -471,7 +471,11 @@ module.exports = function (s, conf) {
                 let to_buy_pct = n(so.buy_pct).subtract(held_pct).value()
                 buy_pct = to_buy_pct
               }
-              fee = so.order_type === 'maker' ? s.exchange.makerFee : s.exchange.takerFee
+              if (so.order_type === 'taker' || buy_pct + s.exchange.takerFee <= 100 && s.exchange.makerBuy100Workaround) {
```

Comment:
I think this makes `taker` the default, vs `maker` what it used to be. Not sure if it matters.

Reply:
You are right and i'd rather not change it, dunno if its relevant though.